### DISCUSSION TITLE
Isn't redis client expected to automatically reconnect?

### DISCUSSION THREAD

**Discussion by azisaka** [kedacore/keda#6241] [Q&A / Need Help]
Hi all! Thank you for the excellent work on Keda! 

I just started to implement the Redis Lists scaler on my cluster and stumbled upon an issue: `"error": "redis: client is closed"`

Some of the Scaled Objects disconnect from Redis and never reconnect. I can induce a reconnection by editing the Scaled Object twice. On the first edit the error shows up on the log; On the second time the client reconnects.

Am I doing anything wrong? Is it a bad approach to automatically reconnect? Do I need to tell anything to Keda to ask for this kind of resilience?

What information would you need to help me figure this out?
Looking forward to contribute here.

**Comment by rickbrouwer** [comment]
which version of Keda is used?

**Reply by zroubalik** [reply]
yeah, good question. The client should reconnect
